Airside_Functions.cpp

```diff
@@ -242,17 +250,16 @@ mavlink_encoding_status_t Mavlink_airside_encoder(POGI_Message_IDs_e msgID, mavl
         {
             one_byte_uint_cmd_t* warg_cmd = (one_byte_uint_cmd_t*) struct_ptr;
 
-            global_position.hdg = warg_cmd ->cmd;
+            global_position.hdg = warg_cmd->cmd;
         }
         break;
 
         default:
             return MAVLINK_ENCODING_BAD_ID;
-            break;
     }
 
     // loading warg command ID before encoding
-    global_position.time_boot_ms = msgID;
+    global_position.time_boot_ms = MESSAGE_ID_BEGIN_LANDING; //use MESSAGE_ID_BEGIN_LANDING for simple testing within airside, otherwise use msgID
```

Comment:
We should switch this back to `msgID` if we only use `MESSAGE_ID_BEGIN_LANDING` for testing.

Reply:
ahh oops thx for catching it!